it/mail/chrome/communicator/wallet/wallet.properties

```diff
@@ -79,5 +79,5 @@ NotConverted = Impossibile convertire i dati archiviati.
 PasswordNotChanged = Impossibile cambiare la password che protegge i tuoi dati sensibili.
 PasswordExpired = I tuoi dati sensibili sono stati messi in sicurezza.
 PasswordNotExpired = Impossibile mettere in sicurezza i tuoi dati sensibili.
-Caveat = Salvare password e altri dati sensibili## Il Gestore password e il Gestore moduli salveranno le password, i nomi utente e altri dati sensibili e li inseriranno per te automaticamente quando saranno richiesti.#Questi dati riservati sono archiviati sul tuo computer in un file che è difficile, ma non impossibile, da leggere.#Se altri utenti hanno accesso al tuo computer, puoi proteggere con password i dati sensibili archiviati scegliendo una Password principale.#Se scegli di proteggere con password i tuoi dati riservati, ti sarà richiesto di fornire di volta in volta la tua Password principale. Questo metodo fornisce una maggiore sicurezza ma è leggermente più scomodo.
+Caveat = Salvare password e altri dati sensibili## Il Gestore password e il Gestore moduli salveranno le password, i nomi utenti e altri dati sensibili e li inseriranno per te automaticamente quando sono richiesti.#Questi dati riservati sono archiviati sul tuo computer in un file che è difficile, ma non impossibile, da leggere.#Se altri utenti hanno accesso al tuo computer, puoi proteggere con password i dati sensibili archiviati scegliendo una Password principale.#Se scegli di proteggere con password i tuoi dati riservati, ti sarà richiesto di fornire di volta in volta la tua Password principale. Questo metodo fornisce una miglior sicurezza ma è leggermente più scomodo.
```

Comment:
CORRECTIONS NEEDED. 

Wrong plural: use "nomi utente" instead of "nomi utenti".